libnetwork/store_test.go

```diff
@@ -83,6 +84,8 @@ func OptionBoltdbWithRandomDBFile(t *testing.T) config.Option {
 }
 
 func TestMultipleControllersWithSameStore(t *testing.T) {
+	testutil.SkipWhenUnprivileged(t)
```

Comment:
Do you know why this one fails?

Reply:
I'm not sure the underlying cause, but it does consistently fail with the following.

```
=== RUN   TestInitializeFromClientHangs
    cli_test.go:218: server never received an init request
--- FAIL: TestInitializeFromClientHangs (1.00s)
```

Reply:
That's a different test.

Reply:
> mkdir /run/docker/libnetwork: permission denied

Reply:
It looks like if we set `ExecRoot` on this oone to something like `t.Tempdir()` that should take care of this.